ios/brave-ios/Sources/Web/Download.swift

```diff
@@ -88,4 +88,16 @@ open class Download: NSObject {
       .union(.illegalCharacters)
     return string.components(separatedBy: validFilenameSet).joined()
   }
+
+  // Truncates the file-name in the middle to the specified maxLength characters.
+  public static func truncateMiddle(fileName: String, maxLength: Int) -> String {
```

Comment:
Should be moved out of `Web` (technically along with `stripUnicode`, was pulled in originally just for legacy reasons during the migration)